src/MySqlConnector/MySqlClient/MySqlSslMode.cs

```diff
@@ -6,7 +6,12 @@
 	public enum MySqlSslMode
 	{
 		/// <summary>
-		/// Do not use SSL. This is the default.
+		/// Use SSL if the server supports it. This is the default.
+		/// </summary>
+		Preferred,
```

Comment:
We should keep the same order as https://github.com/mysql/mysql-connector-net/blob/5864e6b21a8b32f5154b53d1610278abb3cb1cee/Source/MySql.Data/MysqlDefs.cs#L363 so that `default(MySqlSslMode)` is the same (source compatibility). (Even though that's not the same default that the connector would actually use.)